### ISSUE TITLE
Jeff: Please review the website

### ISSUE THREAD

**Issue by TheMapSmith** [VTrans/vtp3-home#21]
http://vtransparency.vermont.gov

Thanks :+1:

**Comment by srsjojo** [comment]
Oops

**Comment by jefframsey** [comment]
Why Transparency?  Transparency is imperative as it not only assures customers that VTrans is effectively managing the resources under its care, but holds VTrans accountable for its actions and decisions.
Should this mention money along with resources or is that taboo?
Contacts links aren't working on the main page.  Contact Form has an error.
http://vtransparency.vtrans.opendata.arcgis.com/#datasets?q=contacts&sort_by=relevance
parenthesis are missing information.  I'll send more as I keep reviewing.

**Comment by jefframsey** [comment]
Open Data links have (from [missing %{name} value])
Who Do I call?  Link to map, the map says "Zoom and click for contacts".  When I zoom, I don't see what to click for contacts.  I figured it out by clicking a town, but should it say "Zoom and click town for contacts" or something like that?  It isn't apparent.

**Comment by jefframsey** [comment]
http://vtransparency.vermont.gov/pavement-metrics.html
On the pavement metrics pages, could the inner views be expanded vertically so I can see some of the information presented below the Details, Table, Charts tabs?  At first glance, it doesn't appear to have information, but if some of the info was showing, as it is on the Projects Maintenance/ Projects Metrics page then I'm more likely to check it out.

**Comment by jefframsey** [comment]
On the Metrics pages, the titles up in the photo of the road are incorrect for various sections.  Pavement Metrics appears twice and Maintenance appears to be on the wrong page.

**Comment by jefframsey** [comment]
On the maps, could the legend automatically open when the map opens?  I understand that I can click on the icon, but I don't think the general public would.  I think about my mom using this site.  She would see pretty colors, but not understand without see the legend displayed.  I don't want to dumb it down, but what about some instruction such as "click on a road/ triangle/ etc for informaiton"?

**Comment by jefframsey** [comment]
On Pavement Metrics page, scroll the inner view down completely and the last two links don't work.  The Open Since 2014 and Powered by GitHub.

**Comment by jefframsey** [comment]
In Project Metrics, Related Datasets, no picture showing.  Some have pics, some don't.  Also, any way to make the Vermont Agency of Transportation logo in the upper left look clearer?  It appears to be fuzzy.

**Comment by jefframsey** [comment]
Maintenance Metrics (Maintenance Activities) charts - donut chart has text written on top of other text and many of the drop down choices are not highlighted for use.

**Comment by jefframsey** [comment]
Overall looks great!  When do we build one for inhouse use so I can put all my permits on it??

**Comment by srsjojo** [comment]
Thanks for all the great feedback, Jeff! 

I've compiled all your comments in #44, so you can see when/if they get addressed.

:star: :star: :star: :star: :star:
